packages/wallets/src/hooks/useSyncLocalStorageClientAccounts.ts

```diff
@@ -22,25 +29,60 @@ const useSyncLocalStorageClientAccounts = () => {
                     excluded_until: data.excluded_until,
                     is_disabled: data.is_disabled,
                     is_virtual: data.is_virtual,
-                    landing_company_name: newAccount.landing_company,
+                    landing_company_name: newAccount.landing_company_shortcode,
```

Comment:
Why we are getting some values from data(activeAccount) and some from newAccount?

Reply:
@mahdiyeh-deriv it's because we receive just these properties in response (image attached) but the rest properties we get from the active wallet because in this time we already switched to just created wallet

<img width="928" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/6996c94a-9a5d-4bcd-87ce-d57dbfbc5305">